src/guides/getting-started.md

```diff
@@ -594,7 +594,7 @@ This barely scratches the surface of what you can do with Diesel, but hopefully
 has given you a good foundation to build off of. We recommend exploring the [API docs] to see more.
 The final code for this tutorial can be found [here][final].
 
-[API docs]: ../docs/index.md
+[API docs]: https://diesel.rs/docs/index
```

Comment:
I'd prefer a relative one, I don't think it's a good way to force the fixed link.

```suggestion
[API docs]: ../docs/index
```